tests/unit/test_client.py

```diff
@@ -47,6 +48,12 @@
     _RetryWithExponentialBackoff,
 )
 
+try:
+    from zoneinfo._common import ZoneInfoNotFoundError  # type: ignore
```

Comment:
Can directly import `zoneinfo.ZoneInfoNotFoundError` - that's the public location.

Reply:
as per https://pypi.org/project/backports.zoneinfo/, with need to import the backport for Python <3.9. Did I miss something?

Reply:
Oh, good point. I was checking with 3.10 only.

Can you add a TODO comment that it can be simplified once we move past 3.9?

Reply:
Sorry I was saying something like this:

```python
try:
    from zoneinfo import ZoneInfoNotFoundError
except ModuleNotFoundError:
    from backports.zoneinfo._common import ZoneInfoNotFoundError
```